src/test/addrman_tests.cpp

```diff
@@ -443,10 +443,19 @@ BOOST_AUTO_TEST_CASE(getaddr_unfiltered)
     CNetAddr source = ResolveIP("250.1.2.1");
     BOOST_CHECK(addrman->Add({addr1, addr2}, source));
 
+    // Tried 3 times and never a success so isTerrible = true
```

Comment:
Could you expand the comment similar to what's above? "The time is set, but after 3 attempts...". Otherwise ACK :)

Reply:
Done!